### ISSUE TITLE
Appium waiting for device to be idle for 10s with every interaction, set it to 100ms or lower

### ISSUE THREAD

**Issue by wulfihm** [appium/appium-desktop#1378]
## The problem

Well the problem is with uiatomator2 actually? In my case it is related to https://github.com/appium/appium/issues/14281
Possibly also related to this: #936

In the Logcat output of the emulator on the uiautomator2.server I am able to see this:
```
2020-05-06 11:35:44.499 24478-24516/io.appium.uiautomator2.server I/appium: Waiting up to 10000ms for device to be idle
2020-05-06 11:35:54.542 24478-24516/io.appium.uiautomator2.server W/QueryController: Could not detect idle state.
```
With everything I do in Appium Desktop it waits 10s. Well at least everytime the app is build such that the phone is not in idle state.
In coded tests (with appium) you can do driver.setSetting(Setting.WAIT_FOR_IDLE_TIMEOUT, 100) but this is not possible in Appium Desktop AFAIK. **So ideally it should be set programmatically in Appium Desktop if possible at all?**

**Comment by KazuCocoa** [comment]
The update setting command is available as below in 1.15.1
![image](https://user-images.githubusercontent.com/5511591/81182175-70732380-8fe8-11ea-948a-fae3acae5a6c.png)

**Comment by KazuCocoa** [comment]
Via capability is like `settings[waitForIdleTimeout]: 1000`
https://appium.io/docs/en/writing-running-appium/caps/
